alembic/versions/46da5adcd1c0_initial_migration.py

```diff
@@ -0,0 +1,49 @@
+"""Initial migration
+
+Revision ID: 46da5adcd1c0
+Revises: 
+Create Date: 2024-11-03 19:59:21.652253
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '46da5adcd1c0'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('city',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=127), nullable=False),
+    sa.Column('additional_info', sa.String(length=511), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
```

Comment:
The unique constraint on the 'name' column in the 'city' table is correctly applied. Ensure that this aligns with your application's requirements, as it will prevent two cities from having the same name.